src/main/resources/application-dev.properties

```diff
@@ -2,10 +2,10 @@ spring.datasource.url=jdbc:postgresql://localhost:26257/defaultdb
 spring.datasource.driver-class-name=org.postgresql.Driver
 spring.datasource.username=root
 spring.datasource.password=
-spring.jpa.hibernate.ddl-auto=update
+spring.jpa.hibernate.ddl-auto=create-drop
```

Comment:
I would leave update here by default - I would prefer to keep my data in the database in between runs

Reply:
I think it might be a good time to start using flyway, because it would allow us to do stuff like update the database. I will look into using it

Reply:
I was testing if the entries are created correctly and wanted to start with a clean slate each time but forgot to change it back to update. App properties now reverted back to the default "develop" state.